impl/src/main/java/com/okta/jwt/impl/jjwt/BaseVerifierBuilderSupport.java

```diff
@@ -16,20 +16,29 @@
 package com.okta.jwt.impl.jjwt;
 
 import com.okta.commons.configcheck.ConfigurationValidator;
+import com.okta.commons.http.authc.DisabledAuthenticator;
+import com.okta.commons.http.config.HttpClientConfiguration;
 import com.okta.jwt.VerifierBuilderSupport;
-import com.okta.jwt.impl.http.OkHttpClient;
+import com.okta.jwt.impl.http.HttpClient;
+import com.okta.jwt.impl.http.OktaCommonsHttpClient;
 import io.jsonwebtoken.SigningKeyResolver;
 
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.time.Duration;
+import java.util.Objects;
 
 abstract class BaseVerifierBuilderSupport<B extends VerifierBuilderSupport, R> implements VerifierBuilderSupport<B, R> {
 
     private String issuer;
     private Duration leeway = Duration.ofMinutes(2);
     private Duration connectionTimeout = Duration.ofSeconds(1);
-    private Duration readTimeout = Duration.ofSeconds(1);
+    private String proxyHost = null;
+    private int proxyPort;
+    private String proxyUsername = null;
+    private String proxyPassword = null;
+    private int maxRetryAttempts = 3;
```

Comment:
Looks like this should be 2 seconds (says the retry logic spec for interactive clients)
@bretterer might be able to confirm

Reply:
`maxRetryAttempts` should be 2 by default based on our spec.